CHANGELOG.md

```diff
@@ -4,6 +4,41 @@
 
 - As of v1.3, support for `brokerList` is deprecated for our Kafka topic scaler and will be removed in v2.0 ([#632](https://github.com/kedacore/keda/issues/632))
 
+## v1.3
```

Comment:
Should we move the `## Deprecations` above to under the v1.3.0 release section in here?